.orchestra/support/revng-container/requirements.txt

```diff
@@ -0,0 +1,12 @@
+aiodataloader
+ariadne
+cffi
+pyelftools
+Jinja2
+hypercorn
+pefile
+python-multipart
+pyyaml
+requests
+requests-toolbelt
+xdg
```

Comment:
Mmmh, can we have this file in `revng`? We should really strive to minimize duplication.

Reply:
I've exposed the `requirements.txt` in revng so that we can drop this one and use that one instead